shared/constants/multichain/networks.ts

```diff
@@ -36,3 +36,7 @@ export const MULTICHAIN_PROVIDER_CONFIGS: Record<
     },
   },
 };
+
+export const MULTICHAIN_NATIVE_CURRENCY_TO_CAIP19 = {
```

Comment:
We should probably introduce a new `assets.ts` for multichain like in my BTC overview PR here: https://github.com/MetaMask/metamask-extension/pull/25200/files#diff-176c069745041e50f5ba4c3b0f903b18a910f48a9c08056ede867c3a6ca0ade7

Reply:
Will be addressed in the overview PR: https://github.com/MetaMask/metamask-extension/pull/25200